src/Console/Edit.php

```diff
@@ -79,8 +79,13 @@ protected function edit($contents)
 
         $process = new Process([config('env-security.editor'), $meta['uri']]);
         $process->setTimeout(null);
-        $process->setTty(true);
+        $process->setTty(Process::isTtySupported());
         $process->mustRun();
+        if (!Process::isTtySupported()) {
+          while (empty(file_get_contents($meta['uri'])) || file_get_contents($meta['uri']) === $contents) {
+            sleep(10);
```

Comment:
10 seconds seems like a potentially long time to wait after saving an ENV change. I'd vote 2 or 3 seconds, I don't think the performance hit of checking the filesystem every 2-3 seconds should be an issue.

Reply:
Agreed :)